main.js

```diff
@@ -309,8 +309,8 @@ app.on('ready', function() {
             .then(function getAccounts() {
                 return ethereumNode.send('eth_accounts', []);
             })
-            .then(function onboarding(result) {
-                if (ethereumNode.isGeth && result && result.length === 0) {
+            .then(function onboarding(resultData) {
```

Comment:
If it throws an error then in the stack trace it will show up, rather than "anonymous function", etc. Plus it makes the code easer to read.